### PR TITLE
Make docs deps match package deps

### PR DESCRIPTION
**Pull request by ryanjulian**

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -13,8 +12,8 @@ pyprind
 python-dateutil
 scikit-image
 scipy
-tensorflow<1.13,>=1.12.0
-tensorflow-probability<0.6.0,>=0.5.0
+# tensorflow<1.16,>=1.15.0
```
Comment (gitanshu) [inline docs/requirements.txt]: Why was this commented? And I guess not providing this would mean we need to add it to mock imports in sphinx?

Comment (ryanjulian) [reply docs/requirements.txt]: I noticed that installing TF makes rtfd.io run out of memory, but now many of the TF and torch modules aren't documented in the auto-generated docs. Not sure what to do about this one.

Comment (krzentner) [reply docs/requirements.txt]: Surely we can't be the only one with this problem?

Comment (krzentner) [reply docs/requirements.txt]: I've opened an issue to address this in the future, so let's merge this change.

**Pull request closed without merging**